stdlib/private/SwiftPrivateLibcExtras/Subprocess.swift

```diff
@@ -264,9 +264,19 @@ public enum ProcessTerminationStatus : CustomStringConvertible {
 
 public func posixWaitpid(_ pid: pid_t) -> ProcessTerminationStatus {
   var status: CInt = 0
+#if CYGWIN
+  withUnsafeMutablePointer(&status) {
```

Comment:
I captured the compiler output. The compiler was patched and built from the snapshot of 7/25.
```
[50/63] Compiling /cygdrive/c/Work/swift/build/Ninja-ReleaseAs...ivate/SwiftPrivateLibcExtras/cygwin/x86_64/SwiftPrivateLibcExtras.
FAILED: cd /cygdrive/c/Work/swift/build/Ninja-ReleaseAssert/swift-cygwin-x86_64/stdlib/private/SwiftPrivateLibcExtras && /usr/bin/cm
. . . . .
/cygdrive/c/Work/swift/swift/stdlib/private/SwiftPrivateLibcExtras/Subprocess.swift:268:20: error: extraneous argument label 'to:' in call
  withUnsafePointer(to: status) {
                   ^~~~~
```